snippets/header-dropdown-menu.liquid

```diff
@@ -5,7 +5,7 @@
   {% render 'header-dropdown-menu' %}
 {% endcomment %}
 
-<nav class="header__inline-menu">
+<nav class="header__inline-menu ">
```

Comment:
Not blocking but if we need to make other changes let's clean up this last bit of whitespace.